zuul.d/projects.yaml

```diff
@@ -96,6 +98,14 @@
       - integrated-queue
       - publish-to-galaxy
 
+- project:
+    name: github.com/ansible-collections/community.aws
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - publish-to-galaxy
+      - publish-to-automation-hub
```

Comment:
we won't publish community to AH, so lets drop this.

Reply:
ah oops, overzealous copy/paste - thx.